<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MCPHA</class>
 <widget class="QMainWindow" name="MCPHA">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Multichannel Pulse Height Analyzer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="cfg" native="true">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <layout class="QGridLayout" name="cfgLayout">
       <item row="0" column="7">
        <widget class="QRadioButton" name="rate_2">
         <property name="text">
          <string>2.5</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QRadioButton" name="rate_0">
         <property name="text">
          <string>0.5</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="addrValue">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>192.168.1.100</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="connectButton">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="rateLabel">
         <property name="text">
          <string>Sample rate, MSPS</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="6">
        <widget class="QRadioButton" name="rate_1">
         <property name="text">
          <string>1.25</string>
         </property>
        </widget>
       </item>
       <item row="0" column="8">
        <widget class="QRadioButton" name="rate_3">
         <property name="text">
          <string>6.25</string>
         </property>
        </widget>
       </item>
       <item row="0" column="9">
        <widget class="QRadioButton" name="rate_4">
         <property name="text">
          <string>12.5</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="addrLabel">
         <property name="text">
          <string>IP address</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="hist1">
       <attribute name="title">
        <string>Spectrum histogram 1</string>
       </attribute>
       <layout class="QGridLayout" name="hist1Layout"/>
      </widget>
      <widget class="QWidget" name="hist2">
       <attribute name="title">
        <string>Spectrum histogram 2</string>
       </attribute>
       <layout class="QGridLayout" name="hist2Layout"/>
      </widget>
      <widget class="QWidget" name="scope">
       <attribute name="title">
        <string>Oscilloscope</string>
       </attribute>
       <layout class="QGridLayout" name="scopeLayout"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
